source/views/dictionary/detail.js

```diff
@@ -63,9 +63,7 @@ export class DictionaryDetailView extends React.PureComponent<Props> {
 
 				<ListFooter
 					href={STO_SA_DICT_URL}
```

Comment:
We should remove this as well

Reply:
For if/when footers ever do become actively linked?

Reply:
Changed.